public/scripts/statusDisplay.js

```diff
@@ -0,0 +1,35 @@
+const displayDiv = document.querySelector(".display-div");
```

Comment:
Does this need to be declared as a module constant? It seems like it's only being used in `updateStatusDisplay`

Reply:
It does not. Before I created the `#updateStatusDisplay` method inside of `main.js`, two different methods made use of `displayDiv`. I didn't realize that only one method makes use of it now.